chompfile.toml

```diff
@@ -81,7 +81,7 @@ deps = [':test:unit', ':test:e2e']
 name = 'test:e2e'
 serial = true
 deps = [':default', ':build']
-run = 'node --loader ./dist/loader.js tests/e2e/test.js'
+run = 'node --loader ./dist/loader.js tests/e2e/test.js -i ../'
```

Comment:
Rather than having to explicitly pass this, I think we should try to ensure that the top-level resolution is performed relative to `process.cwd()` instead of the loader `import.meta.url`. The top-level load can be detected from within a loader based on the `parentURL` being null I believe?